vm-transfer/zvm-transfer.py

```diff
@@ -0,0 +1,494 @@
+        return None
+    return filtered_vm_list[0]
+
+
+def get_vm_by_name(client, vm_name, vm_list=None):
+    if vm_list is None:
+        vm_list = client.vms.list(detailed=True)
+    filtered_vm_list = [vm for vm in vm_list if vm.name == vm_name]
+    if not filtered_vm_list:
+        logger.info("VM with name: %s does not exists", vm_name)
+        return None
+    return filtered_vm_list[0]
+
+
+def init_src_symp_client():
+    my_session = requests.Session()
+    my_session.verify = False
+    client = symphony_client.Client(url='https://%s' % Config.SRC_CLUSTER_IP, session=my_session)
+    client.login(domain=Config.SRC_ACCOUNT,
+                 username=Config.SRC_USERNAME,
+                 password=Config.SRC_PASSWORD,
+                 project='default',
+                 mfa_secret=Config.SRC_MFA_SECRET)
+    return client
+
+
+def init_dst_symp_client():
+    my_session = requests.Session()
+    my_session.verify = False
+    client = symphony_client.Client(url='https://%s' % Config.DST_CLUSTER_IP, session=my_session)
+    client.login(domain=Config.DST_ACCOUNT,
+                 username=Config.DST_USERNAME,
+                 password=Config.DST_PASSWORD,
+                 project='default',
+                 mfa_secret=Config.DST_MFA_SECRET)
+    return client
+
+
+def extract_vm_info(vm):
+    logger.info("Extracting info from VM:\n%s", pformat(unmunchify(vm)))
+    vm_info = dict()
+    vm_info['name'] = vm['name']
+    vm_info['disable_delete'] = vm['disable_delete']
+    vm_info['hw_firmware_type'] = vm['hw_firmware_type']
+    vm_info['instanceType'] = vm['instanceType']
+    vm_info['instance_profile'] = vm['instance_profile']
+    vm_info['key_pair'] = vm['key_pair']
+    vm_info['metadata'] = vm['metadata']
+    vm_info['provided_os_type_id'] = vm['provided_os_type_id']
+    vm_info['ramMB'] = vm['ramMB']
+    vm_info['restart_on_failure'] = vm['restart_on_failure']
+    vm_info['vcpus'] = vm['vcpus']
+    vm_info['ports'] = vm['ports']
+    vm_info['tags'] = [tag for tag in vm['tags'] if not tag.startswith('system:')]
+
+
+    #vm_info['project_id'] = get_dst_project_id(vm['project_id'], src_client, dst_client)
+    #vm_info['bootVolume'] = get_dst_boot_volume_id(vm['bootVolume'], src_client, dst_client)
+    #vm_info['volumes'] = get_dst_volumes_ids(vm['volumes'], src_client, dst_client)
+    #vm_info['imageId'] = get_dst_image_id(vm['imageId'], src_client, dst_client)
+    #vm_info['networks'] = get_dst_networks(vm['networks'], src_client, dst_client)
+    logger.info("vm_info: %s", vm_info)
+    return vm_info
+
+
+def create_networking_map(src_client, dst_client, vm):
+    src_networks = src_client.vpcs.networks.list(project_id=Config.SRC_PROJECT)
+    src_networks_id_to_name = {network.id: network.name for network in src_networks}
+    if len(src_networks) != len(src_networks_id_to_name):
+        msg = "There are at least two networks with the same name in the source"
```

Comment:
if len(src_networks) != len(src_networks_id_to_name) then that would mean there are 2 networks with the same ID, since the key is ID and not name. That is impossible for 2 networks to have the same ID. Therefore this if statement is not needed at all as far as I understand.